src/normalizer.ts

```diff
@@ -76,10 +77,15 @@ export type InternalShape = ObjectBase & {
 
 export type InternalSlideObject = InternalText | InternalImage | InternalShape;
 
+export type InternalImageSrc = {
```

Comment:
This could also be `{ kind: "data", data: string } | { kind: "path", path: string }` to make the attribute a bit nicer for path

Reply:
👍